float32 v_x
float32 v_y
float32 v_z
float32 w_x
float32 w_y
float32 w_z
---
float32 q1_dot
float32 q2_dot
float32 q3_dot
